app/models/pipeline_run_stage.rb

```diff
@@ -390,4 +390,26 @@ def experimental_command
     # Dispatch job
     aegea_batch_submit_command(batch_command, stage_name: DAG_NAME_EXPERIMENTAL, sample_id: sample.id)
   end
+
+  # Gets the URL to the AWS console page of the batch job for display on
+  # admin-only pages.
+  def batch_job_status_url
```

Comment:
This is definitely a perfect case to use view helpers (the true purpose of view). Should not be model methods.

Reply:
Ha, I don't even know what a view helper is. I'll look it up. 

But I still think there is some value for the console or other future callers other than the web page.

Reply:
Sure, lets do it.